pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -619,6 +619,8 @@ type NodeGroup struct {
 	MinSize *int `json:"minSize,omitempty"`
 	// +optional
 	MaxSize *int `json:"maxSize,omitempty"`
+	// +optional
+	MetricsCollection []MetricsCollection `json:"metricsCollection,omitempty"`
```

Comment:
`NodeGroup` is a higher-level construct and it's an implementation detail that it uses an ASG. As such, I think this field's name should reflect that it maps to the provisioned ASG's `MetricsCollection` field, something like `ASGMetricsCollection`. While we do have field names that map directly to the ASG's fields (like the scaling properties), I think some of them make sense at the nodegroup level, while others may have been an oversight.